### ISSUE TITLE
mistake in cpt2.1 'Functions' section

### ISSUE THREAD

**Issue by Messy1** [dvgodoy/PyTorchStepByStep#60]
'
        # Creates the train_step function for our model, 
        # loss function and optimizer
        # Note: there are NO ARGS there! It makes use of the class
        # attributes directly
        self.train_step = self._make_train_step()
        # Creates the val_step function for our model and loss
        self.val_step = self._make_val_step()
'
here the attributes should be 'train_step_fn', 'val_step_fn' instead.

**Comment by dvgodoy** [comment]
Hi @Messy1 

Thank you for pointing this out. You're absolutely correct, these attributes should be named `train_step_fn` and `val_step_fn`. They were correctly named in the "Full Code" section, and now I've fixed them in the "Functions" section.

Thank you for supporting my work!
Best,
Daniel
